lib/store.js

```diff
@@ -26,8 +37,16 @@ Store.prototype.getModel = function(modelName) {
 };
 
 Store.prototype.get = function(modelName, pk) {
+	var key = cacheKey(modelName, pk);
+	if (this.cacheEnabled && this.cache.has(key)) {
+		return this.cache.get(key);
```

Comment:
[Medium] Can we use `Cache.get` directly instead of calling `Cache.has` first and then `Cache.get`? `Cache.get` can use the implementaton`Cache.has` internally.